plugin/pencil.vim

```diff
@@ -98,7 +98,8 @@ if !exists('g:pencil#autoformat_config')
         \   },
         \   'rst': {
         \     'black': [
-        \       'rst(CodeBlock|Directive|LiteralBlock|Sections)',
+        \       'rst(CodeBlock|Directive|ExDirective|LiteralBlock|Sections)',
+        \       'rst(Comment|Delimiter|ExplicitMarkup|SimpleTable)',
```

Comment:
Why use a two line blacklist, why not just add a couple more entries to the existing blaclklist line?

Reply:
It could be one line, but I had broken it into two for readability.

I haven’t had a chance yet to finish testing, but it’s likely safe to merge.

I’d add that rst doesn’t seem to play well with Vim’s autoformat and might benefit from a more sophisticated Syntax module.